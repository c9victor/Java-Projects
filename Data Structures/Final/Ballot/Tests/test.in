100
]
B
L
B
I
A
T
A
S
B
h
A
U
B
V
A
M
B
T
A
p
A
B
A
t
A
P
B
q
A
k
B
V
B
C
B
q
A
B
A
P
B
E
A
M
A
[
B
x
A
i
B
Y
A
h
B
_
B
M
A
d
B
^
B
x
B
k
A
M
B
q
B
U
A
h
A
\
A
`
A
t
B
k
A
D
B
_
A
Y
A
k
B
U
B
B
A
Y
B
r
A
N
B
_
B
P
B
j
A
F
A
O
A
p
B
R
B
M
A
S
A
R
B
Z
B
i
A
x
B
A
A
e
B
d
A
S
B
m
A
V
B
j
A
T
B
T
A
Q
A
t
B
J
B
H
A
d
A
B
A
Z
B
b
A
N
B
\
A
w
A
c
A
k
A
]
A
^
B
M
A
q
A
S
A
_
A
K
A
x
B
U
A
q
A
`
B
M
A
A
B
V
A
