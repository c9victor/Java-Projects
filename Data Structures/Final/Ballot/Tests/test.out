Before recount
Candidate A: 56
Candidate B: 44

After recount
Candidate A: 15
Candidate B: 11

Candidate A wins
